docs/virtual_machines/host-devices.md

```diff
@@ -77,16 +77,13 @@ configuration:
 
  * `pciVendorSelector` is a combination of a `vendor_id:product_id` required for a device identification on a host. This identifier `10de:1eb8` can be found using `lspci`.
 
-	```
-	lspci -nnv|grep -i nvidia
-	65:00.0 3D controller [0302]: NVIDIA Corporation TU104GL [Tesla T4] [10de:1eb8] (rev a1)
-	```
+        lspci -nnv|grep -i nvidia
+        65:00.0 3D controller [0302]: NVIDIA Corporation TU104GL [Tesla T4] [10de:1eb8] (rev a1)
 
  * `mdevNameSelector` is a name of a mediated device type required for a device identification on a host.
-	```
-	For example: mdev type nvidia-226 represents GRID T4-2A
-    	The selector is matched against the content of /sys/class/mdev_bus/$mdevUUID/mdev_type/name
-	```
+
+	For example: mdev type `nvidia-226` represents GRID T4-2A.
+	The selector is matched against the content of `/sys/class/mdev_bus/$mdevUUID/mdev_type/name`.
```

Comment:
This is incorrect.  The correct diff is:

83  * `mdevNameSelector` is a name of a mediated device type required for a device identification on a host.
84
85    For example: 
86    `$ mdev type nvidia-226` represents GRID T4-2A
87
88    The selector is matched against the content of `/sys/class/mdev_bus/$mdevUUID/mdev_type/name`.

Reply:
I'm not sure I understand -- is `mdev` a command?

Reply:
Hm.  Now, I'm not sure.  Would you mind doing a bit of research to see if we have this right?

Reply:
`mdev` is not a command.

Perhaps better phrasing of this language should be:

> For example, mediated device type `nvidia-226` represents `GRID T4-2A`.  The selector is matched against the content of `/sys/class/mdev_bus/$mdevUUID/mdev_type/name`.

FWIW: On my system, `nvidia-226` actually represents `T4-2A`, not `GRID T4-2A`.  I don't know if that's a recent change in the nvidia kubevirt device plugin, or if there's some kind of variation of these labels between different NVIDIA T4 models, `pci.ids` versions, firmware versions, ...or something else...  Regardless, I chose not to remove `GRID` from that label here in this PR.

Reply:
I appreciate your participation in this conversation and your defense of the diff.  :)

If you want to make that selector update you can and I'll accept it.   Otherwise give me a thumbs up and I'll resolve this conversation and we can proceed.

Reply:
No problem!

I just pushed an update with the amended text.  Have a look.